plugins/modules/dcnm_network.py

```diff
@@ -1356,11 +1349,9 @@ def update_ms_fabric(self, diff):
             for node in list_elem['lanAttachList']:
                 node['fabric'] = self.sn_fab[node['serialNumber']]
 
-
     def push_to_remote(self, is_rollback=False):
 
         path = '/rest/top-down/fabrics/{}/networks'.format(self.fabric)
-        bulk_create_path = '/rest/top-down/bulk-create/networks'
```

Comment:
Same comment here. I assume `bulk_create_path` is not used anywhere?

Reply:
This variable "bulk_create_path" is not used anywhere in this method/file after this definition.